packages/ui/package.json

```diff
@@ -50,7 +50,7 @@
     "scripts": {
         "start": "react-scripts start",
         "dev": "react-scripts start",
-        "build": "react-scripts build",
+        "build": "cross-env CI=false react-scripts build",
```

Comment:
what's the purpose of `cross-env CI=false`?

Reply:
if CI=true, reacr-script turn eslint warning into error, failing compilation

in CI environment, CI is set to true

Reply:
https://stackoverflow.com/questions/64518226/my-create-react-app-is-failing-to-compile-due-to-eslint-error

Reply:
another detail
https://stackoverflow.com/questions/50930044/what-does-export-ci-true-do-in-this-create-react-app-deploy-script